beam/www/beam/types/index.ts

```diff
@@ -91,6 +91,7 @@ export type JobCard = ParentDoctype & {
 
 export type StockEntry = ParentDoctype & {
 	items: WorkOrderItem[]
+	docstatus: number
```

Comment:
This can be a part of the `ParentDoctype` type interface since it applies to all doctypes